cli/lib/kontena/cli/stacks/common.rb

```diff
@@ -110,6 +110,11 @@ def require_config_file(filename)
     def generate_volumes(yaml_volumes)
       return [] unless yaml_volumes
       yaml_volumes.map do |name, config|
+        if config['external'].is_a?(TrueClass)
+          config['external'] = name
+        elsif config['external']['name']
```

Comment:
Do we support `external: false`? It seems to pass the options validations and explodes here...

```
NoMethodError: undefined method `[]' for false:FalseClass
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/common.rb:115:in `block in generate_volumes'
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/common.rb:112:in `each'
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/common.rb:112:in `map'
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/common.rb:112:in `generate_volumes'
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/common.rb:79:in `stack_from_reader'
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/common.rb:101:in `stack_read_and_dump'
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/install_command.rb:23:in `execute'
```